test/deploy.js

```diff
@@ -52,7 +52,8 @@ test.beforeEach((t) => {
 test.serial('deployAll', (t) => {
```

Comment:
I might look into batch requests like that depending on the performance situation. This isn't as simple as just doing all interactions all at once: I'd feel more comfortable doing them in batches of 100 or something, giving it a cap. And that's what would bring the extra complexity.

Definitely worth keeping in mind though. I'd prefer to have a separate Issue / PR to try it.